tests/dialects/test_spark.py

```diff
@@ -345,3 +345,13 @@ def test_bool_or(self):
             "SELECT a, LOGICAL_OR(b) FROM table GROUP BY a",
             write={"spark": "SELECT a, BOOL_OR(b) FROM table GROUP BY a"},
         )
+
+    def test_current_user(self):
+        self.validate_all(
+            "CURRENT_USER",
+            write={"spark": "CURRENT_USER()"},
```

Comment:
is this valid? current_user() is a function in spark?

Reply:
It works with `spark-sql` for me:

```sql
➜  sqlglot git:(main) spark-sql -e "SELECT CURRENT_USER()"
Setting default log level to "WARN".
To adjust logging level use sc.setLogLevel(newLevel). For SparkR, use setLogLevel(newLevel).
23/04/06 18:57:55 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
23/04/06 18:57:59 WARN HiveConf: HiveConf of name hive.stats.jdbc.timeout does not exist
23/04/06 18:57:59 WARN HiveConf: HiveConf of name hive.stats.retries.wait does not exist
23/04/06 18:58:02 WARN ObjectStore: Version information not found in metastore. hive.metastore.schema.verification is not enabled so recording the schema version 2.3.0
23/04/06 18:58:02 WARN ObjectStore: setMetaStoreSchemaVersion called but recording version is disabled: version = 2.3.0, comment = Set by MetaStore georgesittas@192.168.0.212
Spark master: local[*], Application Id: local-1680796677723
georgesittas
Time taken: 4.771 seconds, Fetched 1 row(s)
```

Reply:
and what about without paren?

Reply:
Yep, both are aliases for the same functionality, tried the other one as well earlier.